roles/ceph-defaults/defaults/main.yml

```diff
@@ -364,7 +364,7 @@ mds_max_mds: 1
 
 ## Rados Gateway options
 #
-radosgw_frontend_type: beast # For additionnal frontends see: http://docs.ceph.com/docs/nautilus/radosgw/frontends/
+radosgw_frontend_type: beast # For additional frontends see: https://docs.ceph.com/en/octopus/radosgw/frontends/
```

Comment:
```suggestion
radosgw_frontend_type: beast # For additional frontends see: https://docs.ceph.com/en/latest/radosgw/frontends/
```

Reply:
I took this change and re-ran the generate_group_vars_sample.sh script.